sys/auto_init/netif/auto_init_at86rf2xx.c

```diff
@@ -60,6 +60,12 @@ void auto_init_at86rf2xx(void)
         LOG_DEBUG("[auto_init_netif] initializing at86rf2xx #%u\n", i);
 
         at86rf2xx_setup(&at86rf2xx_devs[i], &at86rf2xx_params[i]);
+#ifdef MODULE_GNRC_LWMAC
+        gnrc_netif2_lwmac_create(_at86rf2xx_stacks[i],
+                                 AT86RF2XX_MAC_STACKSIZE,
+                                 AT86RF2XX_MAC_PRIO, "at86rf2xx-lwmac",
+                                 (netdev_t *)&at86rf2xx_devs[i]);
+#else
```

Comment:
Move this into the `#ifdef MODULE_GNRC_NETIF2` below, please

Reply:
Done it~